client/src/pages/Login.js

```diff
@@ -72,16 +71,21 @@ export default function Login(props) {
         setEmailErrMsg("User doesn't exist");
         return;
       }
-      if ("Password does not match") {
+      if (res === "Password does not match") {
```

Comment:
shouldn't this be `res.password === ...`? I recall storing the string on an attribute called password

Reply:
When the interceptor catches errors, it annoyingly doesn't include the body when it gets passed to the receiver, only the error status and some generic error message. I gave your server error responses different statuses and from those statuses I wrote a response to return from there(it's essentially the ones you wrote). So I'm actually returning the statement made from the interceptor on all these error messages.

Reply:
Ahhh okay, gotcha!